src/main/java/org/cactoos/text/BytesAsText.java

```diff
@@ -114,4 +114,9 @@ public String asString() throws IOException {
         return new String(this.bytes.asBytes(), this.charset);
     }
 
+    @Override
+    public int compareTo(final Text text) {
+        return new UncheckedText(text).compareTo(this);
```

Comment:
@fabriciofx A little bit confused order - we compare argument with this , but in original variant is vice versa.  WDYT?